libs/langgraph-checkpoint-aws/README.md

```diff
@@ -338,20 +442,66 @@ def __init__(
                 "kms:DescribeKey"
             ],
             "Resource": "arn:aws:kms:{region}:{account}:key/{kms-key-id}"
-        },
-        {
-            "Effect": "Allow",
-            "Action": [
-                "bedrock:TagResource",
-                "bedrock:UntagResource",
-                "bedrock:ListTagsForResource"
-            ],
-            "Resource": "arn:aws:bedrock:{region}:{account}:session/*"
         }
     ]
 }
 ```
 
+### Valkey Setup (for Valkey components)
+
+#### Using AWS ElastiCache for Valkey (Recommended)
+```python
+# Connect to AWS ElastiCache from host running inside VPC with access to cache
+from langgraph_checkpoint_aws.checkpoint.valkey import ValkeyCheckpointSaver
+
+checkpointer = ValkeyCheckpointSaver.from_conn_string(
+    "valkeys://your-elasticache-cluster.amazonaws.com:6379",
+    pool_size=20
+)
+```
+If you want to connect to cache from a host outside of VPC, use ElastiCache console to setup a jump host so you could create SSH tunnel to access cache locally. 
+
+#### Using Docker
+```bash
+# Start Valkey with required modules
+docker run --name valkey-bundle -p 6379:6379 -d valkey/valkey-bundle:latest
+
+# Or with custom configuration
+docker run --name valkey-custom \
+  -p 6379:6379 \
+  -v $(pwd)/valkey.conf:/etc/valkey/valkey.conf \
+  -d valkey/valkey-bundle:latest
+```
+
+## Performance and Best Practices
+
+### Valkey Performance Optimization
+
+#### Connection Pooling
+```python
+# Use connection pools for better performance
+from valkey.connection import ConnectionPool
+
+pool = ConnectionPool.from_url(
+    "valkey://localhost:6379",
+    max_connections=20,
+    retry_on_timeout=True
+)
+
+with ValkeyCheckpointSaver.from_pool(pool) as checkpointer:
+    # Reuse connections across operations
+    pass
+```
+
+#### TTL Strategy
+```python
+# Configure appropriate TTL values
+checkpointer = ValkeyCheckpointSaver.from_conn_string(
+    "valkey://localhost:6379",
+    ttl_seconds=3600  # 1 hour for active sessions
+)
+```
```

Comment:
Not sure if there is any recommendation here? Is there a default we are setting in the saver class?